doc/zypper.8.txt

```diff
@@ -102,7 +102,16 @@ Throughout this manual we will often refer to resolvables simply as 'packages' a
 
 Package Dependencies
 ~~~~~~~~~~~~~~~~~~~~
-Software packages depend on each other in various ways. Packages usually 'require' or 'recommend' other packages, but they can also 'conflict' with other packages. Packages may support specific hardware or language settings. Zypper uses a 'dependency solver' to find out which packages need to be installed to satisfy the user's request. See https://en.opensuse.org/Libzypp/Dependencies for more information.
+Software packages depend on each other in various ways. Packages usually 'require' or 'recommend' other packages, but they can also 'conflict' with other packages. Packages may support specific hardware or language settings. Zypper uses a 'dependency solver' to find out which packages need to be installed to satisfy the user's request.
```

Comment:
`Packages usually 'require' or 'recommend' other packages, but they can also 'conflict' with other packages.`
->
`Packages usually 'require' or 'recommend' other packages, but they can also 'conflict' with them.`

Reply:
Repeating the `other packages` part seems wrong to me but I'm no native speaker so not sure if that is the case.

Reply:
`with those` might also work...

Reply:
'with them' sounds right to me.